app/Livewire/Project/Application/DeploymentNavbar.php

```
@@ -52,15 +52,24 @@ public function force_start()
 
     public function cancel()
     {
-        $kill_command = "docker rm -f {$this->application_deployment_queue->deployment_uuid}";
+        $deployment_uuid = $this->application_deployment_queue->deployment_uuid;
```

PHPMD (2.15.0)

The variable $deployment_uuid is not named in camelCase. (undefined)

(CamelCaseVariableName)